lib/loader.rb

```diff
@@ -0,0 +1,8 @@
+require 'yaml'
```

Comment:
Rename this file to `new_super_codebreaker_2021.rb`

Reply:
renamed